src/EditorFeatures/CSharp/InlineRename/CSharpEditorInlineRenameService.cs

```diff
@@ -27,57 +22,5 @@ internal sealed class CSharpEditorInlineRenameService : AbstractEditorInlineRena
             : base(refactorNotifyServices, globalOptions)
         {
         }
-
-        protected override bool CheckLanguageSpecificIssues(SemanticModel semanticModel, ISymbol symbol, SyntaxToken triggerToken, [NotNullWhen(true)] out string? langError)
```

Comment:
this code moved, unchanged, to the CSharpRenameIssuesService (defined in features-layer).